Tests/BootCamp.Chapter.Tests/CityTest.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using Xunit;
+using BootCamp.Chapter.Models;
+using BootCamp.Chapter.Exceptions;
+using BootCamp.Chapter.Tests.Public;
+using Moq;
+
+namespace BootCamp.Chapter.Tests
+{
+    public class CityTest
+    {
+        [Theory]
+        [InlineData( 1, -1)]
```

Comment:
Yea i agree but the assignment said:
All heights grid[i][j] are in the range [0, 100].
so that's why the test is there and that's why i added the exception.